generators/app/templates/src/app/screens/InitialLoading/index.ejs

```diff
@@ -0,0 +1,57 @@
+<%_ if (features.onboarding && !features.login) { _%>import { useEffect } from 'react';<%_ } _%>
+import PropTypes from 'prop-types';
+<%_ if (features.login) { _%>
+import { useSelector } from 'react-redux';
+import withLoadable from '@components/Loadable';
+<%_ } _%>
+import Routes from '@constants/routes';
+<%_ if (features.onboarding && !features.login) { _%>
+import { setOnBoardingAccess, getOnBoardingAccess } from './services/onBoardingService';
+<%_ } _%>
+
+<%_ if (features.login) { _%>
+const initialLoadingSelector = state => state.auth.initialLoading;
+<%_ } _%>
+
+const InitialLoading = ({ navigation }) => {
+  <%_ if (features.login) { _%>
+  const initialLoading = useSelector(initialLoadingSelector);
+  const currentUser = useSelector(state => state.auth.currentUser);
+  if (!initialLoading) {
+    <%_ } _%>
+    <%_ if (features.onboarding && !features.login) { _%>
+  useEffect(() => {
+    (async () => {
+      const access = await getAccess();
+      if (!access) setOnBoardingAccess();
+      navigation.navigate(access ? Routes.App : Routes.OnBoarding);
+    })();
+  }, []);
+  const getAccess = () => getOnBoardingAccess();
+  <%_ } _%>
+    <%_ 
+    if(!(features.onboarding && !features.login)){ _%>navigation.navigate(<%_
+    } if (features.login) { _%>currentUser ? Routes.App : Routes.Auth<%_ } 
+    if(!features.onboarding && !features.login){ _%>Routes.App<%_ } if(!(features.onboarding && !features.login)){_%>);<%_ } _%>
```

Comment:
what are you trying to achieve here, because i don't think is fully correct the syntax of this

Reply:
It's correct, I made all the combinations to create the project and it was okey. It's because that file will change a lot depending on what options you have chosen.

Reply:
same question, what's this?

Reply:
it's because sometimes you will create the project with onboarding and without login, and sometimes you will create it with both

Reply:
`if(!(features.onboarding && !features.login))` which case represents this if?